android/kroll-apt/src/java/org/appcelerator/kroll/annotations/generator/ProxyBindingV8.cpp.fm

```diff
@@ -244,12 +244,18 @@ void ${className}::${method.apiName}(const FunctionCallbackInfo<Value>& args)
 	<@Proxy.initMethodID className=className name=name signature=signature logOnly=false/>
 
 	Local<Object> holder = args.Holder();
-	// If holder isn't the JavaObject wrapper we expect, look up the prototype chain
 	if (!JavaObject::isJavaObject(holder)) {
 		holder = holder->FindInstanceInPrototypeChain(getProxyTemplate(isolate));
 	}
-
+	if (holder.IsEmpty() || holder->IsNull()) {
```

Comment:
Move this repeated code chunk into a method/macro? And possibly add some logging when we hit these edge cases that shouldn't be occurring?